data/modules/PolicePatrol/PolicePatrol.lua

```diff
@@ -203,6 +225,14 @@ local onLeaveSystem = function (ship)
 	target = nil
 end
 
+local onShipDocked = function (player)
+	if not player:IsPlayer() then return end
+	local crimes, fine = PlayerState:GetCrimeOutstanding()
```

Comment:
`PlayerState` is a global singleton and should not be self-called. This is inadvertently passing the PlayerState API object as the `faction` parameter.
```suggestion
	local crimes, fine = PlayerState.GetCrimeOutstanding()
```